src/client.ts

```diff
@@ -145,40 +229,59 @@ class StandaloneConnection implements Connection {
     return this._isClosed;
   }
   async execute(query: string): Promise<void> {
```

Comment:
Please add newlines between methods

Reply:
I've added whenever found. But honestly I don't understand why we have a prettifier if it doesn't handle such a basic thing for us.